sdk/python/tests/compiler/testdata/withparam_global.py

```diff
@@ -23,32 +24,63 @@ def empty(self):
 
 TektonCompiler._get_unique_id_code = Coder.empty
 
+op0_yaml = '''\
+name: 'my-out-cop0'
+outputs:
+  - {name: out, type: String}
+implementation:
+    container:
+        image: python:alpine3.6
+        command: ['sh', '-c']
+        args:
+        - |
+          set -e
+          python -c "import json; import sys; json.dump([i for i in range(20, 31)], open(\'\"$0\"\', 'w'))"
```

Comment:
`open(\'\"$0\"\', 'w'))` should be
`open('$0', 'w'))`

Reply:
done